### ISSUE TITLE
The Dockerfile curl a file hosted on omar.yt

### ISSUE THREAD

**Issue by TheFrenchGhosty** [iv-org/invidious#1414]
The Dockerfile curl a file hosted on omar.yt: https://github.com/iv-org/invidious/blob/master/docker/Dockerfile#L11

Since omarroth left invidious this file needs to be moved somewhere else, preferably Github.

**Comment by TheFrenchGhosty** [comment]
So we did some research:

The file is a build from the official [lsquic](https://github.com/litespeedtech/lsquic) library repository.

The weird thing is that this lsquic library is already included in the [lsquic.cr repo](https://github.com/iv-org/lsquic.cr/blob/master/src/lsquic/ext/liblsquic.a) however both files have different hash (but the same content).

We currently don't need to rebuild it to add the [lsquic.cr](https://github.com/iv-org/lsquic.cr/releases/tag/v2.18.1-1) to the docker.

Right now we are looking for someone to test if the file from the lsquic.cr can allow the docker to work correctly:

Invidious needs to be deployed via docker with a modification: 

https://github.com/iv-org/invidious/blob/master/docker/Dockerfile#L11

The URL https://omar.yt/lsquic/liblsquic-v2.18.1.a needs to be replaced with https://raw.githubusercontent.com/iv-org/lsquic.cr/master/src/lsquic/ext/liblsquic.a

In short: both files are supposed to be the same, but they have different hash, we can't figure out why, so we just want to know if the file from the lsquic.cr repo allow the docker deployment to work (if it does we will replace the omar.yt link with it)

**Comment by comradekingu** [comment]
Relying on that server being up, and avoiding potential licensing issues is what I see.

**Comment by SuperSandro2000** [comment]
The file should not be but in the source tree. Binary files do not work great with git and increase the rpeo size drastically.

**Comment by Perflyst** [comment]
We could use https://git-lfs.github.com/ instead

**Comment by SuperSandro2000** [comment]
GitHub has a quite strict quota for lfs files. I think the best solution would be to create releases and attach the binary file there.

**Comment by Perflyst** [comment]
Fixed with https://github.com/iv-org/invidious/commit/c5136ca4d65dd97426101aa68aedc697e3f1df6a
